lib/improver/tests/utilities/test_cube_manipulation.py

```diff
@@ -1709,6 +1710,39 @@ def test_coordinate_raise_exception(self):
                 cube, "realization", raise_exception=True)
 
 
+class Test_enforce_float32_precision(IrisTest):
+    """ Test the enforce_float32_precision utility."""
+
+    def setUp(self):
+        """Create two temperature cubes to test with."""
+        self.cube1 = set_up_temperature_cube()
+        self.cube2 = set_up_temperature_cube()
+
+    def test_basic(self):
+        """Test that the function will return a single iris.cube.Cube with
+           float32 precision."""
+        result1 = self.cube1
+        enforce_float32_precision(result1)
+        self.assertEqual(result1.dtype, np.float32)
```

Comment:
I think usually `assertIsInstance` is used for testing the type of a variable. This applies to the other unit tests here as well.

Reply:
We tried this and it didn't accept np.float32 as an argument. So use origonal code